src/Uno.UI/Microsoft/UI/Xaml/Controls/SplitButton/SplitButtonEventArgs.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.UI.Xaml.Controls;
+
+using SplitButton = Microsoft.UI.Xaml.Controls.SplitButton;
```

Comment:
Why did you need to add those namespaces ?

Reply:
Yes, it wasn't necessary here :-) .